src/components/signin/Mnemonics.js

```diff
@@ -100,23 +117,35 @@ const Mnemonics = ({ screenProps, navigation, styles }) => {
     return [exists, exists && (await userStorage.getProfileFieldDisplayValue('fullName'))]
   }
 
-  const incomingMnemonic = get(navigation, 'state.params.mnemonic', undefined)
```

Comment:
Why would you remove that?
We need to support the option of the mnemonic as parameter from the url

Reply:
well?

Reply:
done [commit](https://github.com/GoodDollar/GoodDAPP/pull/517/commits/81ee2f046ff90026d670ceb3c70c0f2142ed082e)